src/branding/rhel/Makefile.am

```diff
@@ -2,7 +2,6 @@ rhelbrandingdir = $(datadir)/cockpit/branding/rhel
 
 rhelbranding_DATA = \
 	src/branding/rhel/branding.css \
-	src/branding/rhel/bg-login.jpg \
```

Comment:
:+1 to that -- this really looks like an obsolete placeholder -- what's the point in shipping a jpg that is entirely black..